### ISSUE TITLE
Not able to find pretrained checkpoint

### ISSUE THREAD

**Issue by MeruguRanjith** [MishaLaskin/vq-vae#17]
Could you share us any trained checkpoint for reference?
And also, what is the ideal range of your codebook from the trained checkpoint.

When we train this Pytorch code with L1 and perceptual loss, the codebook value range as drastically gone up to -25 to 25.
